### PR TITLE
CNV#34680: RN New virt metric

### PR DESCRIPTION
**Pull request by sjhala-ccs**
Version(s): 4.15

Issue: [CNV-34680](https://issues.redhat.com//browse/CNV-34680)

Link to docs preview: https://71660--ocpdocs-pr.netlify.app/openshift-enterprise/latest/virt/release_notes/virt-4-15-release-notes#virt-4-15-installation-update

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: virt/release_notes/virt-4-15-release-notes.adoc [hunk 1]
```diff
@@ -64,7 +64,9 @@ This release adds new features and enhancements related to the following compone
 
 //CNV-34757: Guest OS console logs are collected by default
 
-//CNV-34680: Metrics update. This is probably related to the note in the Deprecated features section.
+//CNV-34680: Metrics update. 
+* You can use the `kubevirt_vm_created_total` metric (Type: Counter) to query the number of VMs that are created in the specified namespace.
```
Comment (sradco) [inline virt/release_notes/virt-4-15-release-notes.adoc:68]: `that are created` Perhaps `that were created`

Comment (sjhala-ccs) [reply virt/release_notes/virt-4-15-release-notes.adoc:68]: Will it be helpful to link to the [Querying metrics](https://docs.openshift.com/container-platform/4.13/virt/support/monitoring/virt-prometheus-queries.html#about-querying-metrics_virt-prometheus-queries) page in our documentation?

I was thinking we could rewrite the note as:

```suggestion
* You can use Prometheus to monitor the `kubevirt_vm_created_total` metric (Type: Counter). `kubevirt_vm_created_total` returns the total number of VMs created in the specified namespace.
```

**Comment by sjhala-ccs** [comment]
@sradco Can you please respond in Jira to my question about the Kbase article for 4.15 metric changes. I will need to update the [existing release note](https://docs.openshift.com/container-platform/4.14/virt/release_notes/virt-4-14-release-notes.html#virt-4-14-deprecated) for 4.15. Thanks.

File: virt/release_notes/virt-4-15-release-notes.adoc (hunk 1 above)
Comment (sjhala-ccs) [reply virt/release_notes/virt-4-15-release-notes.adoc:68]: Updated the note text

Comment (sjhala-ccs) [reply virt/release_notes/virt-4-15-release-notes.adoc]: @sradco Updated the text per our conversation on Slack. Please review. Thanks!

**Comment by sjhala-ccs** [comment]
@dbasunag Can you please review this? Thanks!

**Review by skopacz1 — COMMENTED** [review]
Just one question, but LGTM

File: virt/release_notes/virt-4-15-release-notes.adoc [hunk 2]
```diff
@@ -64,7 +64,9 @@ This release adds new features and enhancements related to the following compone
 
 //CNV-34757: Guest OS console logs are collected by default
 
-//CNV-34680: Metrics update. This is probably related to the note in the Deprecated features section.
+//CNV-34680: Metrics update 
+* You can now use the `kubevirt_vm_created_total` metric (Type: Counter) to query the number of VMs created in a specified namespace.
```
[the remark this answers is not in the document]
Comment (sjhala-ccs) [reply virt/release_notes/virt-4-15-release-notes.adoc:68]: It's not part of the UI or CLI output. Just the type of metric. I went with the formatting in the [existing docs ](https://docs.openshift.com/container-platform/4.14/virt/monitoring/virt-prometheus-queries.html#virt-promql-vcpu-metrics_virt-prometheus-queries)

**Comment by dbasunag** [comment]
@Ahmad-Hafe can you please review it from the QE side?

**Comment by Ahmad-Hafe** [comment]
shouldnt we release this after https://github.com/kubevirt/kubevirt/pull/11294 is merged ?

**Comment by sradco** [comment]
Its a bug fix, so I dont think it should keep the RN from being merged. The metrics name and details stay the same.

**Pull request merged**